e2e/test/helpers/OutgoingMessageHelper.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+namespace Microsoft.Azure.Devices.E2ETests.Helpers
+{
+    internal static class OutgoingMessageHelper
+    {
+        public static OutgoingMessage ComposeOutgoingTestMessage(out string payload, out string p1Value)
```

Comment:
`OutgoingMessageHelper.ComposeOutgoingTestMessage` is a little more verbose than is needed, methinks.

```suggestion
        public static OutgoingMessage ComposeTestMessage(out string payload, out string p1Value)
```

Reply:
Addressing in the next PR